src/test/java/basic/Exercise05Test.java

```diff
@@ -0,0 +1,14 @@
+package basic;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+public class Exercise05Test {
+    @Test
+    void testFactorial() {
```

Comment:
Viết têm test method như thế này

```suggestion
    void shouldTestFactorial_OK() {
```